docs/submission/dictionary-overview.md

```diff
@@ -1,58 +1,55 @@
 ---
 id: dictionary-overview
-title: Dictionary Overview
+title: Data Dictionary
 ---
 
 The ICGC ARGO [Data Dictionary](/dictionary) expresses the details of the ARGO data model, which adheres to specific formats and restrictions to ensure a standard of data quality. The Data Dictionary defines a set of files, each related to a clinical concept, that can be submitted to the ARGO Data Platform.
 
-To follow the evolution of the data dictionary, check out the [Dictionary Release Notes] (../release-notes/dictionary-releases).
+For recent updates, check the [Dictionary Release Notes](../release-notes/dictionary-releases).
 
-## Understanding the Data Dictionary Table View
+## Understanding the Data Dictionary
 
 The [dictionary table view](/dictionary) lists all of the clinical fields that the ARGO Data Platform accepts, separated by clinical TSV file.
 
-The first list is for the Sample Registration file, which is the only file to be uploaded in the **Register Samples** section. All other lists outline the fields in the clinical TSV files that will be uploaded in the **Submit Clinical Data** section. A list of links on the right side of the page makes it very simple to navigate the clinical files. Each file starts with the name, field count, description, and a file name example to help ensure proper names are used. 
+Field listings can be filtered by Data Tier and Attribute, which can help identify which fields are necessary for [clinical data completion](clinical-data-validation-rules).
 
-The lists can be filtered by Data Tier and Attribute, which can help a data submitter identify which fields are necessary for [clinical data completion](clinical-data-validation-rules). 
+You can explore previous dictionary versions using the dropdown at the top of the dictionary. Using the latest version of the dictionary is required during data submission.
 
-The dictionary version appears in a dropdown above the lists, as well as a date of when it was last updated. You can explore previous dictionary versions by choosing another version number from the dropdown. This dictionary version will also be reflected in the names of the file templates that are downloaded from the Platform. It is important to use the most current files for the clinical validation to run smoothly. 
+### Field Descriptors
 
-### Field Name & Description
-Each field has a name and a description, the name being the same label that appears in the headers of each TSV file. 
+Each field has a data tier and an attribute classification, which reflects the importance of the field in terms of clinical data completion.
 
-### Data Tier & Attributes 
-Each field has a data tier and an attribute classification, which reflects the importance of the field in terms of clinical data completion. 
+![ID](/assets/submission/dictionary-id.svg) attribute indicates:
```

Comment:
ID is a data tier not an attribute

Reply:
changes to classification

Reply:
like should it be the order they appear in the dictionary? Data tier to attribute?
ID
Core
Extended
Required
Conditional